File: app/src/main/java/com/neome/api/meta/base/dto/EntVdSendMessageToUsers.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class EntVdSendMessageToUsers extends EntVdAutoStep
{
  @Nullable
  public StudioValueParagraph message;

  @Nullable
  public StudioBuildArgBinder sender;

  @Nullable
  public StudioDtoUserFilter toUsers;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdSendMessageToUsers.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStep
import com.neome.api.meta.base.dto.StudioBuildArgBinder
import com.neome.api.meta.base.dto.StudioDtoUserFilter
import com.neome.api.meta.base.dto.StudioValueParagraph

interface EntVdSendMessageToUsers : EntVdAutoStep
{
  val message: StudioValueParagraph?
  val sender: StudioBuildArgBinder?
  val toUsers: StudioDtoUserFilter?
}